readthedocs/projects/admin.py

```diff
@@ -135,50 +136,79 @@ class ProjectSpamThreshold(admin.SimpleListFilter):
 
     def lookups(self, request, model_admin):
         return (
+            (
+                self.NOT_ENOUGH_SCORE,
+                _("Not enough score (1-{})").format(
```

Comment:
"Not enough score" sounds like something other than "not spam". Any reason we don't just want to call these projects "not spam"?

```suggestion
                _("Not spam (1-{})").format(
```

Reply:
Agreed on this change 👍